hack/code-gen/gen.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
What is this file used for? It is not mentioned in any doc or code

Reply:
@adityabhatia I think you introduce this back in the day because of lister code gens. Could you elaborate here?